test/common/http/dispatcher_test.cc

```diff
@@ -127,32 +127,32 @@ TEST_F(DispatcherTest, BasicStreamHeadersOnly) {
                      .counter("internal.upstream_rq_200")
                      .value());
 
-  // Ensure that the callbacks on the observer were called.
+  // Ensure that the callbacks on the callbacks were called.
```

Comment:
nit: this sentence is a bit weird now. Do you mind updating here and everywhere in the file?